Nos/Views/Components/ActivityView.swift

```diff
@@ -5,11 +5,19 @@ struct ActivityViewController: UIViewControllerRepresentable {
 
     var activityItems: [Any]
     var applicationActivities: [UIActivity]?
+    var completion: (() -> Void)? // Completion handler called after the share activity is finished
```

Comment:
Could we make this a doc comment and simplify it slightly? Something like this (mostly copied from [Apple's docs](https://developer.apple.com/documentation/uikit/uiactivityviewcontroller/1622022-completionwithitemshandler)):

```suggestion
    /// The completion handler to execute after the share activity is finished.
    var completion: (() -> Void)? 
```